modules/scripts/startup-script/templates/startup-script-custom.tpl

```diff
@@ -29,7 +29,7 @@ stdlib::runner() {
   case "$1" in
     ansible-local) stdlib::run_playbook "$destpath/$filename" "$args";;
     # shellcheck source=/dev/null
```

Comment:
What is this line doing? Do we still need `source=/dev/null` now that you are using `.`?

Reply:
Removed